ostree-compliance-mode.spec

```diff
@@ -0,0 +1,58 @@
+%global dracutdir %(pkg-config --variable=dracutdir dracut)
+
+Name:           ostree-compliance-mode
+Version:        0.1
+Release:        1%{?dist}
+Summary:        ostree-compliance-mode is a tool designed to purge protected data, leaving the system in a modifyable state in compliance with GPLv3
+
+License:        GPLv2
+URL:            https://github.com/ericcurtin/%{name}
+Source0:        %{url}/archive/%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires: meson
+BuildRequires: gcc
+BuildRequires: ninja-build
+BuildRequires: ostree
+BuildRequires: rpm-ostree
+BuildRequires: python3-devel
+
+%description
+%{summary}
+
+%prep
+%autosetup
+%global debug_package %{nil}
+
+%build
+%meson
+%meson_build
+
+%check
+%meson_test
+
+%install
+%meson_install
+meson build --buildtype=release --prefix=$RPM_BUILD_ROOT%{_prefix};
+ninja -v -C build
+ninja -v -C build install
+chmod 4011 $RPM_BUILD_ROOT%{_bindir}/ostree-compliance-mode
+
+mkdir -p $RPM_BUILD_ROOT%{_prefix}/libexec
+cp ostree-compliance-mode-helper $RPM_BUILD_ROOT%{_prefix}/libexec/ostree-compliance-mode-helper
+chmod 100 $RPM_BUILD_ROOT%{_prefix}/libexec/ostree-compliance-mode-helper
+
+mkdir -p $RPM_BUILD_ROOT/etc
+currently_deployed_ref="fedora:fedora/35/aarch64/silverblue"
+echo "$currently_deployed_ref" > $RPM_BUILD_ROOT/etc/ostree-compliance-mode.conf
```

Comment:
We cannot hardcode things like `fedora/35/aarch64` here what we can do in the %post section rather than the install section is determine the currently deployed ref and populate the file. It could be aarch64, it could be x86_64, it could be fedora, it could be centos stream, etc.

It might be a good idea to leave population of /etc/ostree-compliance-mode.conf to a separate PR after this though. For version 0.1-2 maybe...

Reply:
Yep, agreed, that was just temporarily hard-coded while I was still working on the other pieces. I've taken the ostree-compliance-mode.conf piece out for now as you suggested and will come back it. Thanks Eric!